app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,20 @@
+<h1><%= @recipe.name %></h1>
+<h2>Complexity: <%= @recipe.complexity %></h2>
+<h2>Genre: <%= @recipe.genre %></h2>
+<% @recipe_ingredients.each do |recipe_ingredient| %>
```

Comment:
Like I said elsewhere, this could also be written as 
```
@recipe.ingredients.each do |ingredient|
```
That way you're accessing ingredient objects directly, and don't need to do `recipe_ingreidnet.ingredient.name` below.